pynicotine/plugins/core_commands/__init__.py

```diff
@@ -290,11 +288,12 @@ def help_command(self, args, user=None, room=None):
                 "query": search_query
             }
 
-        for group_name, commands in command_groups.items():
+        for group_name, command_data in command_groups.items():
             output_text += f"\n\n{group_name}:"
 
-            for command_usage, description in commands:
-                output_text += f"\n\t{command_usage}  -  {description}"
+            for command, aliases, parameters, description in command_data:
+                command_message = f"/{', /'.join([command] + aliases)} {' '.join(parameters)}"
+                output_text += f"\n\t{command_message}  -  {description}"
```

Comment:
It might as well be combined onto one line...
`f"\n\t/{', /'.join([command] + aliases)} {' '.join(parameters)}  -  {description}"`

Reply:
Won't work if you want to avoid the extra space when no parameters are available.

Reply:
Restored strip() for command_message.